package.json

```diff
@@ -15,20 +15,19 @@
   "devDependencies": {
     "chai": "^4.1.2",
     "del": "^1.2.1",
+    "fibers": "^2.0.0",
     "eslint": "^4.17.0",
     "eslint-config-google": "^0.9.1",
     "gas-local": "^1.3.0",
-    "gulp": "^3.9.0",
+    "gulp": "^3.9.1",
     "gulp-bump": "^0.3.1",
```

Comment:
While you're cleaning up deps, `gulp-bump` doesn't seem to be used.

Reply:
Good catch!